admin_guide/topics/creating-LC-and-ASG-cluster-auto-scaler.adoc

```diff
@@ -0,0 +1,103 @@
+// Module included in the following assemblies:
+//
+// * admin_guide/cluster-autoscaler.adoc
+
+[id='creating-LC-and-ASG-cluster-auto-scaler-{context}']
+= Creating Launch Configuration and Auto Scale Group
+
+Before we can run the cluster-autoscaler deployment we need to create
+a LC and an ASG that reference a primed image. The LC that we create
+must be configured so that on first boot the new node will
+automatically join the existing cluster.
+
+.Procedure
+
+. Create the *_bootstrap.kubeconfig_* by copying it from the master node
++
+[source,bash]
+----
+ssh master "sudo cat /etc/origin/master/bootstrap.kubeconfig" > ~/bootstrap.kubeconfig
+----
+
+. Encode *_bootstrap.kubeconfig_* using base64(1)
++
+[source,bash]
+----
+base64 ~/bootstrap.kubeconfig
+XBpVmVyc2lvbjogdjEKY2x1c3RlcnM6Ci0gY2x1c3RlcjoKICAgIGNlcnRpZmljYXRlLWF1dGhv
+cml0eS1kYXRhOiAuLi4=
+...
+----
+
+. Create the *_user-data.txt_* cloud-init stanza
++
+[source]
```

Comment:
Remove "[source]" if your'e not going to specify a type.

Reply:
> Remove "[source]" if your'e not going to specify a type.

Done.